scripts/microbe_editor/microbe_editor.lua

```diff
@@ -54,6 +54,7 @@ function MicrobeEditor:activate()
 end
 
 function MicrobeEditor:update(renderTime, logicTime)
+    -- self.nextMicrobeEntity being a temporary used to pass the microbe from game to editor TODO replace with species fiddling
```

Comment:
Hey wait a minute, why didn't this microbe-passing code run in MicrobeEditor:activate()? Seems like that would've made more sense for something meant to run on gamestate-entry.